README.md

```diff
@@ -133,32 +61,31 @@ Thank you to everyone contributing to Tauri!
 
 ### Documentation
 
-Documentation in a polyglot system is a tricky proposition. To this end, we prefer to use inline documentation of Rust code and at JSDoc in typescript / javascript code. We autocollect these and publish them using Docusaurus v2 and netlify. Here is the hosting repository for the documentation site: https://github.com/tauri-apps/tauri-docs
-
-### Testing & Linting
+Documentation in a polyglot system is a tricky proposition. To this end, we prefer to use inline documentation of Rust code and JSDoc in TypeScript / JavaScript code. We autocollect these and publish them using [starlight](https://starlight.astro.build/) and [netlify](https://netlify.com). Here is the hosting repository for the documentation site: https://github.com/tauri-apps/tauri-docs
 
-Test all the things! We have a number of test suites, but are always looking to improve our coverage:
+## Partners
 
-- Rust (`cargo test`) => sourced via inline `#[cfg(test)]` declarations
-- Typescript (`jest`) => via spec files
-- Smoke Tests (run on merges to latest)
-- eslint, clippy
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="middle">
+        <a href="https://crabnebula.dev" target="_blank">
+          <img src=".github/sponsors/crabnebula.svg" alt="CrabNebula" width="283">
+        </a>
+      </td>
+    </tr>
+  </tbody>
+</table>
 
-### CI/CD
-
-We recommend you read this article to understand better how we run our pipelines: https://www.jacobbolda.com/setting-up-ci-and-cd-for-tauri/
+For the complete list of sponsors please visit our [website](https://tauri.app#sponsors) and [Open Collective](https://opencollective.com/tauri).
```

Comment:
There's also sponsors to be found on Github: https://github.com/sponsors/tauri-apps
Maybe we should list that as well.

Open Collective only shows the monthly payouts so you'd have to look on Github to see a breakdown of who the sponsors are.

Reply:
That's true, i tried to get it in at first but my english failed me on that. Maybe we can cover that with Lorenzo's sponsor review at which point we will probably remove the opencollective mention anyway. Also, github sponsors are visible on the repo so it's that's something.

Anyway, if you can come with a good wording that doesn't sound like i wrote it i'm happy to add it as well.